test/unit/sdam/monitoring.test.js

```diff
@@ -97,7 +96,7 @@ describe('monitoring', function() {
     });
   });
 
-  describe('Monitor', function() {
+  describe('Monitor class', function() {
```

Comment:
The convention has been to `describe('ClassName', ...)` without further modifiers. Does this context help in your mocha side bar?

Reply:
Oh yea whoops, because this test shares the same name as an import it was doing funky things to the discovery, I didn't mean to keep this
![image](https://user-images.githubusercontent.com/10873993/87686573-a5bb9080-c752-11ea-8384-0777b060aa7d.png)